site/index.html

```diff
@@ -6,7 +6,9 @@
   <div class="container">
     <div class="row align-items-center">
       <div class="col-6 mx-auto col-md-6 order-md-2">
-        <img class="img-fluid mb-3 mb-md-0" src="{{ site.baseurl }}/docs/{{ site.docs_version }}/assets/img/bootstrap-stack.png" alt="" width="1024" height="860">
```

Comment:
Do we have the original image? Maybe an svg is smaller here?

Reply:
I do not have the original image, otherwise it'd definitely be smaller.

Reply:
@mdo: do you happen to have this SVG?